cfgov/cfgov/settings/base.py

```diff
@@ -792,11 +792,12 @@
 # middleware. This list of regular expressions defines a set of URLs against
 # which we don't want this logic to be run.
 PARSE_LINKS_EXCLUSION_LIST = [
-    # Wagtail admin pages, except preview and draft views
+    # Wagtail admin pages, except preview, draft, and debug views
     (
         r"^/admin/(?!"
         r"pages/\d+/(edit/preview|view_draft)/|"
-        r"mega_menu/menu/preview/\w+/"
+        r"mega_menu/menu/preview/\w+/|"
+        r"template_debug/"
```

Comment:
Per the comment just above this line, this regex excludes "Wagtail admin pages, _except_ preview, draft, and debug views" (emphasis added). Normally we don't apply our middleware to admin URLs because we don't want e.g. external link icons getting added in Wagtail. But we do want the middleware to run on some admin URLs, including the template debug view.

Reply:
D'oh! Reading comprehension fail; sorry.